tests/components/autocomplete_dropdown.test.ts

```diff
@@ -412,3 +445,20 @@ describe("autocomplete boolean functions", () => {
     }
   );
 });
+describe("composer entries", () => {
+  test("Autocomplente entries are sorted by length and then alphanumerically", async () => {
```

Comment:
We should ideally mock the function registry for all the tests, or else the test will break when we introduce a new function. We shoudl create a helper so we can do it easily in one line tho, something like


```js
mockFunctionRegistry(["SEC", "SUM", "SLN", ...]);
```

Reply:
I agree on the idea, but you'd have 2 ways to do thing then. you either mock it with existing functions or you define new functions for the sake of the test (what we currently do).

Reply:
Do we ever need to mock it with existing functions ? :thinking:

Reply:
maybe not :o